src/App.vue

```diff
@@ -48,5 +48,6 @@ onMounted(async () => init());
       </div>
     </div>
   </template>
-  <BaseFlashNotification />
+  <TheFlashNotification />
+  <TheModalNotification />
```

Comment:
Why `The` keyword 😄 any reasons?

Reply:
@ChaituVR https://vuejs.org/style-guide/rules-strongly-recommended.html#single-instance-component-names